GodotEnv/src/features/addons/commands/install/AddonsInstallCommand.cs

```diff
@@ -20,6 +21,13 @@ public class AddonsInstallCommand : ICommand, ICliCommand {
   )]
   public int? MaxDepth { get; init; }
 
+  // If we have any top-level addons that are symlinks, we know we're going
+  // to need to elevate on Windows.
+  public bool IsWindowsElevationRequired =>

```

Comment:
I opened the PR to refactor/win-elevation.

Maybe I can add a commit to this PR to do something like that in RunElevatedOnWindows:
```
bool shouldElevate = 
      Debugger.IsAttached ||
      new WindowsPrincipal(WindowsIdentity.GetCurrent())
          .IsInRole(WindowsBuiltInRole.Administrator);

    Process process = new() {
      StartInfo = new() {
        FileName = exe,
        Arguments = args,
        UseShellExecute = shouldElevate,
        Verb = shouldElevate ? "runas" : string.Empty,
        CreateNoWindow = !shouldElevate
      }
    };
```

With this code, elevation will occur in RunElevatedOnWindows if the command was not already elevated.

And this code can be removed:
```
if (
      !Debugger.IsAttached &&
      !new WindowsPrincipal(WindowsIdentity.GetCurrent())
        .IsInRole(WindowsBuiltInRole.Administrator)
    ) {
      throw new InvalidOperationException(
        "RunElevatedOnWindows is only supported with admin role."
      );
    }
```

Reply:
`new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator)`

`new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator)` is also the statement used in ShouldElevateOnWindows/IsElevatedOnWindows. It is true if
- GodotEnv is executed in a Windows shell that has been run as administrator (so there is no need to elevate in any case)
- GodotEnv has been re-executed following a call to ElevateOnWindows

The modification I have proposed for ElevateOnWindows will have the following effect

- The user is not prompted to elevate if GodotEnv already has administrator right (if the Windows shell already has admin rights or if the command was elevated because IsWindowsElevationRequired is true)
- The user is prompted to elevate if, for some reason, ElevateOnWindows is called and GodotEnv does not have administrator rights (for example during debugging or for your particular case concerning symlink addons)

The shouldElevate definition can even be simplified in the code above, as `new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator)` is false when debugging:
bool shouldElevate = 
      !(new WindowsPrincipal(WindowsIdentity.GetCurrent())
          .IsInRole(WindowsBuiltInRole.Administrator));

I noticed that I'd forgotten the "!" character in my previous reply, sorry if it was misleading.

Reply:
I tried to disable UAC in the Windows settings. 

When UAC is disabled, no prompt is displayed (and no additional window is opened). I've also tested with the current version (1.6) of GodotEnv, and no prompt is displayed when UAC is disabled either, so there doesn't seem to be any change in behaviour.